assessmentmodel-sdk/src/main/java/org/sagebionetworks/bridge/assessmentmodel/upload/AssessmentResultArchiveUploader.kt

```diff
@@ -90,7 +93,13 @@ abstract class AssessmentResultArchiveUploader(
             assessmentResult.runUUIDString
         }
 
-        val uploadFile = persist(assessmentRunUUID, builder.build(), sessionWindowExpiration)
+        val uploadMetadata: Map<String, JsonElement> = mapOf(
+            "instanceGuid" to JsonPrimitive(assessmentInstanceId),
+            "eventTimeStamp" to JsonPrimitive(eventTimestamp)
```

Comment:
```suggestion
            "eventTimestamp" to JsonPrimitive(eventTimestamp)
```
? Or does the key capitalization not matter (or did I get it wrong on iOS)?

Reply:
Good catch. I thought I had just copied from iOS.